### ISSUE TITLE
Complete implementation of `cilium identity get` 

### ISSUE THREAD

**Issue by ianvernon** [cilium/cilium#1278]
`cilium identity get` currently only works for reserved identities (host and world), and does not output any information about existing identities:

```  vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium endpoint list
ENDPOINT   POLICY        IDENTITY   LABELS (source:key[=value])   IPv6                     IPv4            STATUS   
           ENFORCEMENT                                                                                              
38939      Enabled       258        container:id.baz              fd02::c0a8:210b:0:981b   10.11.250.189   ready    
56326      Enabled       257        container:id.bar              fd02::c0a8:210b:0:dc06   10.11.129.91    ready    
                                    container:id.teamA                                                              
62006      Enabled       256        container:id.foo              fd02::c0a8:210b:0:f236   10.11.116.202   ready    
                                    container:id.teamA                                                              
vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium identity get --list
host              1
world             2
vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium identity get
Error: Invalid identity ID

Retrieve the identity of the specified label

Usage:
  cilium identity get [flags]

Flags:
      --list   List all identities

Global Flags:
      --config string   config file (default is $HOME/.cilium.yaml)
  -D, --debug           Enable debug messages
  -H, --host string     URI to server-side API
vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium identity get container:id.baz
vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium identity get id.baz          
vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium identity get 10.11.250.189
vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium identity get world        
2
vagrant@cilium-master:~/go/src/github.com/cilium/cilium/tests$ cilium identity get host 
1

```

`cilium identity get` is meant to resolve all identities, but it has not been fully implemented.

**Comment by ashwinp** [comment]
PR: https://github.com/cilium/cilium/pull/1313

**Comment by ianvernon** [comment]
Merged #1313
